gitless/cli/gl_fetch.py

```diff
@@ -21,3 +22,5 @@ def parser(subparsers, _):
 
 def main(args, repo):
     core.git_wrap('fetch', *args.fetch_args)
+    pprint.ok('Fetching from upstream succeeded')
```

Comment:
The fetch could fail for any number of reasons which will be printed to stdout/stderr. We should check the `returncode` from the `git_wrap()` call and only print the ok message if `returncode == 0`. Without the check the user will always see a success message which may not be true.

Same applies to:
- [ ] gl_pull.py
- [ ] [gl_patch.py](https://github.com/goldstar611/gitless/pull/33/files#diff-e7155535a5cadd32eeb2bb38c5fcc31228792b6005b7b6c64fef2f2f29efd65d)